### DISCUSSION TITLE
Win32 build?

### DISCUSSION THREAD

**Discussion by Cunya** [NVIDIA-Omniverse/PhysX#132] [Q&A] [answered]
Is it possible to build PhysX 5 for Win32? Or get the binaries from somewhere?

**Comment by ayoub-belarbi** [comment, marked as answer]
Hi @Cunya,
Unfortunately We removed Win32 support in PhysX 5.
If you absolutely need it, it shouldn't be that hard to make it work (create the necessary presets, adjust the cmake files, check how it's done in PhysX 4 etc..), however we can't help with any build issues.
Thanks
